pages/blog/slack-bot_with_cloudflare.mdx

```diff
@@ -0,0 +1,145 @@
+import withArticle from '../../ui/blog/article'
+export const meta = {
+  title: 'Building Slack bot with Cloudflare Workers',
+  author: 'tuvalSimha',
+  tags: ['graphql', 'yoga', 'codegen'],
```

Comment:
```suggestion
  tags: ['graphql', 'codegen'],
```
The Slack bot doesn't use Yoga. I'm also thinking about a new category for such things, maybe `edge` or `tutorials`? It kind of touches GraphQL and Codegen but the main topic here is CF Worker and its scheduler feature.